opentelemetry-appender-tracing/src/layer.rs

```diff
@@ -114,6 +114,23 @@ impl<LR: LogRecord> tracing::field::Visit for EventVisitor<'_, LR> {
             .add_attribute(Key::new(field.name()), AnyValue::from(value));
     }
 
+    fn record_u64(&mut self, field: &tracing::field::Field, value: u64) {
+        #[cfg(feature = "experimental_metadata_attributes")]
+        if is_duplicated_metadata(field.name()) {
+            return;
+        }
+        match i64::try_from(value) {
```

Comment:
nit: Use `if let Ok()` construct instead of `match` for a cleaner less indented code.

Reply:
done.